src/politech_backend/settings.py

```diff
@@ -121,12 +121,15 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'politech',
-        'USER': os.environ["DB_USER"],
-        'PASSWORD': os.environ["DB_PASS"],
-        'HOST': '127.0.0.1',
+        'NAME': 'postgres',
```

Comment:
Can this be reverted back to `politech`?

Possibly by using https://github.com/veritus/veritus-backend/pull/48/files#diff-354f30a63fb0907d4ad57269548329e3L24